core/src/main/java/bio/terra/pearl/core/service/survey/SurveyResponseService.java

```diff
@@ -255,31 +258,39 @@ public List<Answer> createOrUpdateAnswers(List<Answer> answers, SurveyResponse r
         List<Answer> updatedAnswers = answers.stream().map(answer -> {
             Answer existing = existingAnswerMap.get(answer.getQuestionStableId());
             if (existing != null) {
-                return updateAnswer(existing, answer, response, survey, ppUser, changeRecords);
+                return updateAnswer(existing, answer, response, survey, ppUser, changeRecords, operator);
             }
-            return createAnswer(answer, response, survey, ppUser);
+            return createAnswer(answer, response.getSurveyResponse(), survey, ppUser);
```

Comment:
I think we should we be creating change records on create as well - I could see it being a not uncommon flow for admins to fill out surveys for users on their behalf over the phone, so having a record of that seems important.

Reply:
I see - I was just surprised when I made changes and only SOME of them were reflected in the data audit table, and it took me a while to figure out that it was because only updates are audited. I thought my changes weren't going through / there was a bug, and I could see users feeling the same way.